Src/WitsmlExplorer.Frontend/components/Modals/SettingsModal.tsx

```diff
@@ -27,10 +27,17 @@ const timeZoneLabels: Record<TimeZone, string> = {
 
 const SettingsModal = (): React.ReactElement => {
   const {
-    operationState: { theme, timeZone, colors, dateTimeFormat },
+    operationState: { theme, timeZone, colors, dateTimeFormat, decimals },
     dispatchOperation
   } = useContext(OperationContext);
 
+  const [checked, updateChecked] = useState<string>(() => {
+    const storedDecimalPreference = localStorage.getItem(STORAGE_DECIMAL_KEY);
+    return storedDecimalPreference === DecimalPreference.Raw ? DecimalPreference.Raw : DecimalPreference.Decimal;
```

Comment:
I see that the value is read from localStorage in `index.tsx` when first opening the page. Is there any reason to also get the value from localStorage here?